src/lib.rs

```diff
@@ -1,6 +1,15 @@
+#![feature(plugin)]
+#![cfg_attr(test, plugin(stainless))]
+
+#[cfg(feature = "mock")] extern crate taxonomy_mock as taxonomy;
+#[cfg(not(feature = "mock"))] extern crate foxbox_taxonomy as taxonomy;
```

Comment:
I had to create a cargo feature because integration tests run against a regular debug build (and not a test one). Hence, I can't use `#[cfg(test)]`